lib/dry_twitter/views/sign_in.rb

```diff
@@ -0,0 +1,22 @@
+require "dry_twitter/view/controller"
+
+module DryTwitter
+  module Views
+    class SignIn < DryTwitter::View::Controller
+      configure do |config|
+        config.template = "sign_in"
+      end
+    end
+
+    def locals(options = {})
+      if options.size > 0
+        super.merge(
+            user: options[:input]["user"],
+            errors: options[:input][:errors]
```

Comment:
Mixing string and symbol guys is a great way to create confusion.

Reply:
I'm aiming to use symbols whenever they can be used. In this instance the stringed key is coming from the framework.